transformation/src/main/scala/org/broadinstitute/monster/hca/HcaPipelineBuilder.scala

```diff
@@ -256,6 +262,53 @@ object HcaPipelineBuilder extends PipelineBuilder[Args] {
   def encode(msg: Msg): String =
     upack.transform(msg, StringRenderer()).toString
 
+  def validateJson(filenamesAndMsg: SCollection[(String, Msg)]): Unit = {
+    // pull out the url for where the schema definition is for each file
+    val content = filenamesAndMsg.map {
+      case (filename, msg) => (msg.read[String]("describedBy"), (filename, msg))
+    }
+    // get the distinct urls (so as to minimize the number of get requests) and then get the schemas as strings
+    val schemas = content.map(_._1).distinct.map(url => (url, requests.get(url).text))
```

Comment:
OH, other thing to check: Try running this branch in Dataflow to make sure `requests` and the circe stuff are all serializable. If not, you'll get a runtime error.

Reply:
ran it locally, it worked

Reply:
Can you link me to the dataflow job? So I can sanity-check the graph